src/js/bootstrap-datetimepicker.js

```diff
@@ -122,6 +122,10 @@
                 18: 'alt',
                 'space': 32,
                 32: 'space',
+                'end': 35,
+                35: 'end',
+                'home': 36,
+                36: 'home',
```

Comment:
https://www.w3.org/TR/wai-aria-practices/#datepicker suggests that the `end` button _Moves to the last day of the current month._ and `home` button _Moves to the first day of the current month._.
I didn't add those as defaults, as they did not exist before.